MH4U Dex/HomeViewController.m

```diff
@@ -28,6 +30,21 @@ - (IBAction)resetButtonPressed:(id)sender
     [CoreDataController setShouldTriggerReloadUponRestart:YES];
 }
 
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    [MBProgressHUD showHUDAddedTo:self.view animated:YES];
+    dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
+    dispatch_async(queue, ^{
+        [[CoreDataController sharedCDController] tryLoadSequence];
```

Comment:
If that's the case, then what do you suggest I should do?  In the best case scenario, where a prebuilt-store is being used, the only time managedObjectContext is being called here is when it is being set to nil.  But if a prebuilt store isn't being used, then all of the various JSON-loading methods can potentially get called by tryLoadSequence.   How would I work with all of those in a background-thread context?  And would I still be able to access them later with a main-thread context?